crates/core/src/ibc.rs

```diff
@@ -70,3 +73,92 @@ impl FromStr for IbcTxDataRefs {
         serde_json::from_str(s)
     }
 }
+
+/// The target of a PGF payment
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    Ord,
+    Eq,
+    PartialOrd,
+    BorshDeserializer,
+    Hash,
+)]
+pub struct PGFIbcTarget {
```

Comment:
I moved this here from gov (`crates/governance/src/storage/proposal.rs`) as it's being used from ibc actions - it's mostly just for passing could fields with borsh encoding